.freight.experimental.yml

```diff
@@ -0,0 +1,22 @@
+sentry:
+  organization: sentry
+  project: snuba
+  repository: getsentry/snuba
+
+steps:
+  - kind: KubernetesDeployment
+    selector:
+      label_selector: service=snuba
+    containers:
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: generic-metrics-sets-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: generic-metrics-distributions-consumer
+  - kind: KubernetesCronJob
+    selector:
+      label_selector: service=snuba
+    containers:
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: cleanup
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: optimize
```

Comment:
Why are optimize and cleanup here ?

Reply:
I don't know what these do, what would happen if this step gets executed both in the regular consumers deployment and the experimental deployment?

I will remove it, just thought it was necessary when copying over the config.

Reply:
They can be removed, they are for the cron jobs that automatically clean up Clickhouse partitions.